Makefile

```diff
@@ -135,8 +135,11 @@ check: $(NODE_MODULES_TEST) $(VM_IMAGE) test/common
 
 # checkout Cockpit's bots for standard test VM images and API to launch them
 # must be from master, as only that has current and existing images; but testvm.py API is stable
+# support CI testing against a bots change
 bots:
-	[ -d bots ] || git clone --depth=1 https://github.com/cockpit-project/bots.git
+	git clone --quiet --reference-if-able $${XDG_CACHE_HOME:-/nonexisting}/cockpit-project/bots https://github.com/cockpit-project/bots.git
```

Comment:
I'm fine with this as-is, but according to https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html the default value for XDG_CACHE_HOME should be ~/.cache. 

It's not that important and I know I didn't do this in other places in cockpit myself, but I want to make a PR to bots to change this.

Reply:
Good point, thanks! Updated. Let's do this correctly.